tests/src/core/physics/CharacterController.test.ts

```
@@ -5,150 +5,266 @@
-  it("test constructor", () => {
+  it("constructor", () => {
     // Test that the constructor works correctly.
     const controller = roleEntity.getComponent(CharacterController);
-    expect(controller.stepOffset).to.equal(0.5);
-    expect(controller.nonWalkableMode).to.equal(ControllerNonWalkableMode.PreventClimbing);
-    expect(controller.upDirection).to.deep.include({ x: 0, y: 1, z: 0 });
-    expect(controller.slopeLimit).to.equal(0.707);
+    expect(controller.stepOffset).eq(0.5);
+    expect(controller.nonWalkableMode).eq(ControllerNonWalkableMode.PreventClimbing);
+    expect(controller.upDirection).deep.include({ x: 0, y: 1, z: 0 });
+    expect(controller.slopeLimit).eq(0.707);
```

Biome

[error] Prefer constants from the standard library.

Unsafe fix: Use Math.SQRT1_2 instead.


(lint/suspicious/noApproximativeNumericConstant)